Gemfile.lock

```diff
@@ -26,9 +26,9 @@ GEM
       slop (~> 3.4)
     rainbow (2.0.0)
     rake (10.4.2)
-    rubocop (0.31.0)
+    rubocop (0.33.0)
```

Comment:
Is this update intentional? Does it fix or change some aspect of config handling?